quinn-proto/src/lib.rs

```diff
@@ -150,12 +149,7 @@ pub mod fuzzing {
 }
 
 /// The QUIC protocol version implemented
-const VERSION: std::ops::RangeInclusive<u32> = 0xff00_001d..=0xff00_0020;
-
-/// Whether a 4-byte slice represents a supported version number
-fn is_supported_version(x: &[u8]) -> bool {
-    VERSION.contains(&u32::from_be_bytes(x.try_into().unwrap()))
-}
+const DEFAULT_VERSION: std::ops::RangeInclusive<u32> = 0xff00_001d..=0xff00_0020;
```

Comment:
This name implies a single version, when it's actually a range. Maybe `DEFAULT_SUPPORTED_VERSIONS`?